printer-linter/src/printerlinter/formatters/def_json_formatter.py

```diff
@@ -1,29 +1,84 @@
 import json
 import re
+from collections import OrderedDict
 from pathlib import Path
+from typing import Dict
 
 from .formatter import FileFormatter
 
+
+# Dictionary items with matching keys will be sorted as if they were the value
+# Example: "version" will be sorted as if it was "0"
+TOP_LEVEL_SORT_PRIORITY = {
+    "version":  "0",
+    "name":     "1",
+    "inherits": "3"
```

Comment:
```suggestion
    "inherits": "3",
```
add a trailing comma, to limit the line changes in future diffs, if sort keys are added